lib/broadside/ecs/ecs_deploy.rb

```diff
@@ -170,21 +156,21 @@ def update_task_revision
 
     # reloads the service using the latest task definition
     def update_service
-      task_definition_arn = EcsManager.get_latest_task_definition_arn(family)
-      debug "Updating #{family} with scale=#{@target.scale} using task_definition #{task_definition_arn}..."
+      task_definition_arn = EcsManager.get_latest_task_definition_arn(@target.family)
+      debug "Updating #{@target.family} with scale=#{@scale} using task_definition #{task_definition_arn}..."
 
       update_service_response = EcsManager.ecs.update_service({
         cluster: @target.cluster,
-        desired_count: @target.scale.to_i,
-        service: family,
+        desired_count: @scale,
```

Comment:
i think you want to keep the `to_i` just to guarantee it's a integer, given `ActiveModel`'s predilection for claiming `"1234"` is an integer (hey, mysql agrees!)